.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -141,6 +141,7 @@ object CalypsoApps: BuildType({
 			name = "Build artifacts"
 			scriptContent = """
 				set -x
+				export IS_CI=true
```

Comment:
Essentially just defined this for everything that ultimately builds calypso apps.